spec/models/user_option_spec.rb

```diff
@@ -18,6 +18,28 @@
 
   end
 
+  describe ".mailing_list_mode" do
```

Comment:
`.` is used for class methods. Since `mailing_list_mode` is an instance method, `#` should be used instead.